hw/ip/kmac/doc/checklist.md

```diff
@@ -129,8 +129,8 @@ Coverage      | [SIM_COVERAGE_MODEL_ADDED][]          | Done        |
 Code Quality  | [TB_LINT_SETUP][]                     | Done        |
 Integration   | [PRE_VERIFIED_SUB_MODULES_V1][]       | Done        |
 Review        | [DESIGN_SPEC_REVIEWED][]              | Done        |
-Review        | [DV_PLAN_TESTPLAN_REVIEWED][]         | Not Started |
-Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Not Started |
+Review        | [DV_PLAN_TESTPLAN_REVIEWED][]         | Done        |
+Review        | [STD_TEST_CATEGORIES_PLANNED][]       | Done        |
```

Comment:
I saw most other IPs have exceptions here, such as power, security, and debug, I wonder if that also applies for KMAC?

Reply:
ah good point! you're right, there are no plans to do these testing in the UVM DV environment, will update accordingly.